preprocessors/ner/clipscore.py

```diff
@@ -0,0 +1,343 @@
+'''
```

Comment:
For my own note, is this entirely taken from CLIPscore directly?

Reply:
The code is from clipscore, but I had to make some changes to:
1. combine it into one single python script
2. make it usable within other python scripts, rather than through the command line.